added duo + solo kk necro rots

SalteaRS opened the pull request:
# PvME Change Submission
Thank you for helping maintain our resources! Below is a quick sanity checklist that helps make sure we can keep track of what is changing. Please fill it out :)

## Sanity Checks
- [x] - The title of this pull request clearly describes the change I would like to make.
- [x] - If there are multiple changes in this pull request, they are all related.
- [x] - I have tried to write clear a commit message(s) that describes the changes I made.

Crisnyc commented:
-Necro preset link seems to link to the base duo preset
-In the duo section, the only difference between the base and dps seems to be the pocket slot
-Implement the previous conclusion regarding the drops section, so remove it
-The relics section could probably also be removed if we link an image to the presets in the preset generator link embed

SalteaRS commented:
> -Necro preset link seems to link to the base duo preset -In the duo section, the only difference between the base and dps seems to be the pocket slot -Implement the previous conclusion regarding the drops section, so remove it -The relics section could probably also be removed if we link an image to the presets in the preset generator link embed

I'm only adding the necro part. idk why the preset was a melee one, I fixed it. Seems the preset maker is buggy at times but I fixed it.
